convert2rhel/checks.py

```diff
@@ -389,3 +393,9 @@ def _bad_kernel_substring(kernel_release):
         )
         return True
     return False
+
+
+def check_environment():
+    if "CONVERT2RHEL_NOT_SUPPORTED" in os.environ:
```

Comment:
We should use `CONVERT2RHEL_UNSUPPORTED`
```suggestion
    if "CONVERT2RHEL_UNSUPPORTED" in os.environ:
```

Reply:
because?

Reply:
I'm fine with unsupported. It's shorter and has the same meaning :)

Reply:
The prefix of our vars doesn't say we are seeking for shorter vars:) what about C2R_UNSUPPORTED?

Reply:
I expect the variable to be used not only by developers but also by customers. So far we haven't used the abbreviated form C2R or c2r anywhere publicly and I'd refrain from doing so for professional reasons.

Reply:
Part of the issue it was agreed to use `CONVERT2RHEL_UNSUPPORTED`. It's also similar to leapp's `LEAPP_UNSUPPORTED`. Makes it more consistent 

Currently you have logging that say `Variable CONVERT2RHEL_UNSUPPORTED has been detected` as well